src/dialogs/locales/fr.json

```diff
@@ -164,7 +164,115 @@
     "phoneNumberFormatMaxErrorMsg": "Il semble que les numéros de téléphone que vous avez entrés n'utilisent pas le bon format. Veuillez vérifier votre numéro de téléphone avant de continuer. Voulez-vous réessayer?",
     "rateRetryMsg": "Oops - I'm not sure what you meant. Can you rate the service you received today?",
     "standardQueryMsg": ["Oui s'il vous plaît", "Non merci"],
-    "NoNotificationMsg": "D'accord, c'est bon. Vous avez tout alors!"
+    "NoNotificationMsg": "D'accord, c'est bon. Vous avez tout alors!",
+
+
+    "always-on-bot-block-start": "-------ALWAYS-ON-BOT-START------",
```

Comment:
we should pick a format - camelCase or dash-delimited

Reply:
this is just to highlight the always on bot statements

Reply:
i vote for camelCase